### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by n0099**

### CODE REVIEW

**Comment by ota-meshi** [comment]
Could you please explain what this PR improves?

File: README.md [hunk 1]
```diff
@@ -70,13 +70,8 @@ npm install --save-dev astrojs-compiler-sync@latest @astrojs/compiler
 ## 📖 Usage
 
 1. Change the `include` in your `tsconfig.json` to include the component files (`*.vue`, `*.svelte`, and `*.astro`).
-2. Write `overrides.parserOptions.parser` option into your `tsconfig.json` file.
```
Comment (n0099) [inline README.md:73]: It should be `.eslintrc.cjs`

File: README.md [hunk 2]
```diff
@@ -87,113 +82,89 @@ npm install --save-dev astrojs-compiler-sync@latest @astrojs/compiler
...
+                "project": "./tsconfig.json"
             },
-            // ....
         },
         {
             "files": ["*.vue"],
             "parser": "vue-eslint-parser",
             "parserOptions": {
-                "parser": require("typescript-eslint-parser-for-extra-files"),
+                "parser": "typescript-eslint-parser-for-extra-files",
```
Comment (n0099) [inline README.md:101]: it's the same

File: README.md [hunk 3]
```diff
@@ -87,113 +82,89 @@ npm install --save-dev astrojs-compiler-sync@latest @astrojs/compiler
...
                 // Or
                 // "parser": {
-                //     "ts": require("typescript-eslint-parser-for-extra-files")
+                //     "ts": require("typescript-eslint-parser-for-extra-files"),
```
Comment (n0099) [inline README.md:104]:
```suggestion
                //     "ts": require("typescript-eslint-parser-for-extra-files")
```

File: README.md [hunk 4]
```diff
@@ -87,113 +82,89 @@ npm install --save-dev astrojs-compiler-sync@latest @astrojs/compiler
...
                 // }
-                "project": "./your/tsconfig.json"
-                // ....
+                "project": "./tsconfig.json"
             },
-            // ....
```
Comment (n0099) [inline README.md:117]: too many placeholders

File: README.md [hunk 5]
```diff
@@ -87,113 +82,89 @@ npm install --save-dev astrojs-compiler-sync@latest @astrojs/compiler
...
 ```
 
 ## 👻 Limitations
 
-- Angle bracket type assertions cannot be used.
+- Angle bracket type assertions cannot be used:
 
-  This parser parses as JSX enabled.  
-  Therefore, angle bracket type assertions cannot be used. Use the `as` operator instead.
+  This parser parses as JSX enabled, therefore angle bracket type assertions cannot be used. Use the `as` operator instead.
```
Comment (n0099) [inline README.md:167]: merge sentence

**Review by ota-meshi — APPROVED** [review]
Thanks!

**Pull request merged**